TeamHangman/TeamHangman/GameLetter.cpp

```diff
@@ -0,0 +1,73 @@
+#include "GameLetter.h"
+
+
+
+GameLetter::GameLetter(sf::Vector2f position, std::string letter, int textSize, sf::Color textColor)
+{
+	window = HWindow::GetWindow();
+	label = new Label(position, letter, textSize, textColor);
+	bottomLine = new sf::RectangleShape(sf::Vector2f(30.0f, 3.0f));
+	bottomLine->setPosition(sf::Vector2f(label->GetGlobalBounds().left + label->GetGlobalBounds().width * 0.5f, label->GetGlobalBounds().top + label->GetGlobalBounds().height));
+	bottomLine->setFillColor(sf::Color::White);
+	myLetter = letter;
+}
+
+GameLetter::GameLetter()
+{
+	window = HWindow::GetWindow();
+	label = new Label();
+	bottomLine = new sf::RectangleShape(sf::Vector2f(50.0f, 10.0f));
+	myLetter = "null";
+}
+
+GameLetter::~GameLetter()
+{
+	if (label != nullptr)
+	{
+		delete label;
+	}
+	if (bottomLine != nullptr)
+	{
+		delete bottomLine;
+	}
+}
+
+void GameLetter::Update()
```

Comment:
Are we going to use this empty Update function?

Reply:
probably not